CHANGELOG.md

```diff
@@ -53,6 +55,13 @@ waitForLaunch
   * `mobile: listXCTestsInTestBundle` to return the list of XCTest bundles
 * Add audio recording for simulators and real devices [appium-xcuitest-driver#1207](https://github.com/appium/appium-xcuitest-driver/pull/1207)
   * Read [Audio Capture From iOS Simulators and Real Devices](https://appium.io/docs/en/writing-running-appium/ios/audio-capture/)
+* Add `WebDriverAgentRunner-Runner.app` for simulators in `appium-webdriveragend` npm package to improve build time [WebDriverAgent#331](https://github.com/appium/WebDriverAgent/pull/331)
```

Comment:
I'd rather say `Added prebuilt WebDriverAgentRunner snapshots to speed up tests execution on iOS Simulator`